src/components/speiling/sykmeldinger/sykmelding/sykmeldingOpplysninger/flereopplysninger/UtdypendeOpplysninger.tsx

```diff
@@ -33,22 +33,17 @@ const UtdypendeOpplysninger = (
   const { sykmelding } = utdypendeOpplysningerProps;
   const utdypendeOpplysninger = sykmelding.utdypendeOpplysninger;
   return (
-    <>
-      {utdypendeOpplysninger && (
-        <div className="sykmeldingSeksjon">
-          <h4 className="sykmeldingSeksjon__tittel">{texts.title}</h4>
+    utdypendeOpplysninger && (
```

Comment:
Ser flere steder at vi bruker` ternary ? <>noe</> : null`, mens her `boolean && <></>`. Er det noen forskjell?

Reply:
Noen steder går det greit å returnere en `boolean` imens andre steder må vi returnere `null`. Disse
```
x && y && <Komponent />
```
kan returnere `x` eller `y` om disse er falsy. De vil da returnere verdien de er. Så hvis `x` er en `boolean` vil den returnere `false`.
I dette tilfellet er ikke en `boolean` et gyldig `JSX element` og kan derfor ikke renderes i `FlereOpplysninger.tsx` . Derfor må vi flere steder bruke ternary og eksplisitt returnere `null` fremfor `boolean`.
`utdypendeOpplysninger` er av type `Map` så er usikker på om det noen gang vil bli falsy. Kanskje om det er et tomt `Map` 🤔

Reply:
Aha riktig! Så det er fordi vi har typet den komponenten at den klager? For det er nok mange steder vi ikke har typet komponenten, men bruker `bool && komponent` der dette kan forekomme?

Reply:
Tror ikke fordi vi har typet komponenten. En `boolean` kan være en del av kroppen til et `JSX.Element`, men kan ikke _være_ et `JSX.Element`.
Skjønner ikke helt selv når det fungerer og ikke🤔